bug_reports/Bash_JSONPath.sh.md

```diff
@@ -61,7 +61,7 @@ The following queries provide results that do not match those of other implement
   ```
   Error:
   ```
-  ggrep: Invalid range end
+  grep: Invalid range end
```

Comment:
You'll see some change like that, due to Linux and OSX being slightly different. Safe to ignore/commit.

Nobody's been bothered enough so far to address this :)